src/pages/token-metadata/pnfts.md

```diff
@@ -30,9 +30,9 @@ Let's dive into these two abilities in more detail.
 
 ## More granular delegates
 
-Since all PNFTs operations must go through the Token Metadata program, it can create a new delegate system on top of the spl-token delegate. One that is more granular and allows PNFT owners to pick the operations they want to delegate to a third party.
+Since all pNFTs operations must go through the Token Metadata program, it can create a new delegate system on top of the spl-token delegate. One that is more granular and allows pNFT owners to pick the operations they want to delegate to a third party.
 
-Information for this new delegate system is stored on a special **Token Record** PDA that is derived from both the Mint and the Token accounts of the PNFT. When a new delegate authority is assigned to a PNFT, the Token Metadata program synchronizes that information on both the Token account and the Token Record account.
+Information for this new delegate system is stored on a special **Token Record** PDA that is derived from both the Mint and the Token accounts of the pNFT. When a new delegate authority is assigned to a pNFT, the Token Metadata program synchronizes that information on both the Token account and the Token Record account.
```

Comment:
Thank you for fixing all these!  Great cleanup, getting the docs nicer and nicer each iteration.